### PR TITLE
data: zaidahmad25

### PR DESCRIPTION
**Pull request by zaidahmad25**

### CODE REVIEW

File: data/zaidahmad25.json [hunk 1]
```diff
@@ -24,93 +21,61 @@
...
     {
-      "title": "Learned JavaScript from Udemy",
-      "date": "Oct, 2022 - Dec 13, 2022",
-      "icon": "SiUdemy",
-      "color": "yellow",
-      "description": "After years of exploring, finally decided to go into web development",
-      "url": "https://www.udemy.com/certificate/UC-2b08ae8d-5413-432b-9d84-ec0163e79437/"
+     "title": "Started learning React",
+      "date": "May 2, 2023",
+      "icon": "SiReact",
```
Comment (ChinmayMhatre) [inline data/zaidahmad25.json:73]:
```suggestion
      "icon": "SiReact"
```

**Review by ChinmayMhatre — CHANGES_REQUESTED** [review]

**Comment by ChinmayMhatre** [comment]
Looks great! Added a comment regarding an extra comma which is failing the test

File: data/zaidahmad25.json [hunk 2]
```diff
@@ -1,17 +1,14 @@
 {
   "name": "Zaid Ahmad",
-  "type": "personal",
-  "bio": "Empowering growth and learning through my passion for web development and open source ❤️",
+  "type": "public",
```
Comment (ChinmayMhatre) [inline data/zaidahmad25.json:3]: We no longer use the type property. You can remove this!

**Review by ChinmayMhatre — CHANGES_REQUESTED** [review]

**Comment by zaidahmad25** [comment]
How do I delete this PR? 

I'll make a new PR with the right changes

**Review by ChinmayMhatre — CHANGES_REQUESTED** [review]

File: data/zaidahmad25.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (zaidahmad25) [reply data/zaidahmad25.json:3]: Oh, I did not know about that. My bad 😅

**Comment by ChinmayMhatre** [comment]
@zaidahmad25  it is okay for now, but you do not need to create a new PR you can push the changes in the same branch and they are reflected here!

**Pull request closed without merging**